addons/point_of_sale/static/src/js/models.js

```diff
@@ -2464,7 +2464,9 @@ exports.Orderline = Backbone.Model.extend({
             price_extra: this.get_price_extra(),
             customer_note: this.get_customer_note(),
             refunded_orderline_id: this.refunded_orderline_id,
-            price_manually_set: this.price_manually_set
+            price_manually_set: this.price_manually_set,
+            refunded_orderline_id: this.refunded_orderline_id,
+            refundedQty: this.refundedQty,
```

Comment:
I don't think this is needed, can you check ?
`refunded_qty` is a compute field in the backend